packages/graphile-build-pg/src/utils.js

```diff
@@ -44,3 +44,33 @@ export const camelCase = formatInsideUnderscores(camelCaseAll);
 export const constantCase = formatInsideUnderscores(constantCaseAll);
 export const upperCamelCase = (str: string): string =>
   upperFirst(camelCase(str));
+
+export const parseTags = (str: string) => {
+  return str.split(`\n`).reduce((prev, curr) => {
+    if (prev.text !== "") {
+      return Object.assign({}, prev, {
+        text: `${prev.text}\n${curr}`,
+      });
+    }
+    const match = curr.match(/^@[a-zA-Z][a-zA-Z0-9_]*($|\s)/);
+    if (!match) {
+      return Object.assign({}, prev, {
+        text: curr,
+      });
+    }
+    const key = match[0].substr(1).trim();
+    const value = match[0] === curr ? true : curr.replace(match[0], "");
+    return Object.assign({}, prev, {
+      tags: Object.assign({}, prev.tags, {
+        [key]: !prev.tags.hasOwnProperty(key)
+          ? value
+          : prev.tags[key].constructor === Array
```

Comment:
This should just be `Array.isArray(prev.tags[key])`; this has been supported since Node 0.x; e.g.